index.d.ts

```diff
@@ -125,6 +125,10 @@ declare module "hyperapp" {
     effecter: (dispatch: Dispatch<S>, payload: P) => void | Promise<void>,
     payload: P
   ]
+  
+  // Effects can be declared conditionally
+  type MaybeEffect<S, P> = null | false | true | undefined | "" | 0 | Effect<S, P>
```

Comment:
It works but we can tighten it up a little:
```js
type MaybeEffect<S, P> = null | undefined | "" | 0 | boolean | Effect<S, P>
```

Reply:
Ah of course 👍

Reply:
Fixed in force-push